editor/animation_bezier_editor.cpp

```diff
@@ -1083,7 +1083,8 @@ void AnimationBezierTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 					if (I.key == REMOVE_ICON) {
 						if (!read_only) {
 							EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-							undo_redo->create_action("Remove Bezier Track");
+							undo_redo->create_action("Remove Bezier Track", UndoRedo::MERGE_DISABLE, this);
+							undo_redo->force_fixed_history();
```

Comment:
```suggestion
							undo_redo->create_action("Remove Bezier Track", UndoRedo::MERGE_DISABLE, animation.ptr());
```
`force_fixed_history()` is unnecessary in this case.

Reply:
So it is, thank you!